src/OpenSearch.Client/README.md

```diff
@@ -0,0 +1,3 @@
+`OpenSearch.Client` is a high level OpenSearch .NET client that still maps very closely to the original OpenSearch API. All requests and responses are exposed through types, making it ideal for getting up and running quickly.
```

Comment:
Perhaps rather than -
`All requests and responses are exposed through types, making it ideal for getting up and running quickly.`
something more like -
`All requests and responses are exposed through types, making it easy for users to get up and running quickly.`